src/contrib/cluster/Akka.Cluster.Sharding/Serialization/ClusterShardingMessageSerializer.cs

```diff
@@ -140,6 +152,10 @@ public override byte[] ToBinary(object obj)
                 case ShardRegion.StartEntityAck o: return StartEntityAckToProto(o).ToByteArray();
                 case Shard.GetShardStats o: return new byte[0];
                 case Shard.ShardStats o: return ShardStatsToProto(o).ToByteArray();
+                case GetShardRegionStats o: return new byte[0];
```

Comment:
I think, .NET standard offers `Array.Empty<byte>()` which is static and doesn't allocate new object every time.

Reply:
Not supported in .NET 4.5.2, which we are still multi-targeting. I'm thinking about dropping multi-targeting altogether but I don't want to byte that off in the first beta of v1.4.0 just yet.